### ISSUE TITLE
KeyError: 'welcome-message' on run

### ISSUE THREAD

**Issue by jacobtomlinson** [opsdroid/opsdroid#236]
If `welcome-message` is not specified in the configuration at all then opsdroid fails to start.

This error needs to be caught in a try/except.

```
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/jacob/Projects/opsdroid/opsdroid/opsdroid/__main__.py", line 130, in <module>
    main()
  File "/Users/jacob/Projects/opsdroid/opsdroid/opsdroid/__main__.py", line 123, in main
    welcome_message(opsdroid.config)
  File "/Users/jacob/Projects/opsdroid/opsdroid/opsdroid/__main__.py", line 93, in welcome_message
    if config['welcome-message']:
KeyError: 'welcome-message'
```

**Comment by jacobtomlinson** [comment]
@FabioRosado

**Comment by FabioRosado** [comment]
I should had thought about that!

**Comment by accessdenied12** [comment]
@jacobtomlinson - Wouldn't it be easier to just use an if/in?  I have this ready to submit.  Or I can do a try/except.  Would you like the except to just pass?

**Comment by jacobtomlinson** [comment]
Fixed in #239
